iotdb-client/cli/src/main/java/org/apache/iotdb/cli/IoTDBSyntaxHighlighter.java

```diff
@@ -67,12 +67,14 @@ public AttributedString highlight(LineReader reader, String buffer) {
 
   @Override
   public void setErrorPattern(Pattern errorPattern) {
-    IoTPrinter.print("");
+    // TODO: Not quite sure what this code should actually do
```

Comment:
This class was temporarily disabled in [PR](https://github.com/apache/iotdb/blob/master/iotdb-client/cli/src/main/java/org/apache/iotdb/cli/utils/JlineUtils.java#L90).

Perhaps we can remove this class, or leave it at that? Adding these comments does not seem to help

Reply:
Yeah ... I added comments to code that I didn't understand ... in this particular case I couldn't see what the code actually did, so I added a comment.